docs/tutorial/quick-start.md

```diff
@@ -47,6 +47,9 @@ In Electron, we have provided the [ipc](../api/ipc-renderer.md) module for
 communication between the main process and renderer process. There is also a
 [remote](../api/remote.md) module for RPC style communication.
 
+And finally there are different techniques [to share objects between web
+pages](../faq/electron-faq.md#) of the same window or of different windows.
```

Comment:
Should the `#` after `electron-faq.md` include the section name of this new FAQ so that the link takes you right there?